src/chttpd/src/chttpd.erl

```diff
@@ -1134,14 +1134,14 @@ maybe_handle_error(Error) ->
     case chttpd_plugin:handle_error(Error) of
         {_Code, _Reason, _Description} = Result ->
             Result;
+        {shutdown, Err} ->
+            exit({shutdown, Err});
         {Err, Reason} ->
             {500, couch_util:to_binary(Err), couch_util:to_binary(Reason)};
         normal ->
             exit(normal);
-        {shutdown, Err} ->
-            exit({shutdown, Err});
-        Error ->
-            {500, <<"unknown_error">>, couch_util:to_binary(Error)}
+        Error1 ->
```

Comment:
When I see `Error1` I think maybe it's a modification of the original `Error`, but all the other case clauses use `Err`. Would that perhaps be easier to read?

Reply:
Agree. Good idea.